src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -148,168 +170,184 @@ const std::string LightName(const smart_objects::SmartObject& light_name) {
   return ok ? name : "unknown";
 }
 
-bool CheckLightDataByCapabilities(
+mobile_apis::Result::eType GetItemCapability(
     const smart_objects::SmartObject& capabilities,
-    const smart_objects::SmartObject& light_data) {
+    const std::map<std::string, std::string>& mapping,
+    const std::string& request_parameter,
```

Comment:
examples of the input request_parameter? kStatus/kDensity/kSRGBColor, but not kId?

Reply:
@yang1070 yes, for light module it will be  kStatus/kDensity/kSRGBColor, but not kId